### ISSUE TITLE
May I ask if this library support kotlin language?

### ISSUE THREAD

**Issue by youyuanme** [LitePalFramework/LitePal#329]
![1](https://user-images.githubusercontent.com/11382521/34068984-cbcb0aca-e281-11e7-873b-f5d9d6109adc.png)
![2](https://user-images.githubusercontent.com/11382521/34068985-cc011930-e281-11e7-8326-a46e273b23a4.png)
![3](https://user-images.githubusercontent.com/11382521/34068987-cc3747d0-e281-11e7-855a-562b37252601.png)
Using the kotlin language to create bean objects, the table structure is different from the imagination.

**Comment by guolindev** [comment]
Of course. LitePal support Kotlin completely. But you just need to modify your codes a little bit.

For your model class. Do not use **val**, use **var** instead. Cause using **val** which declared the field as final, thus LitePal couldn't assign any value to it by reflection.

**Comment by youyuanme** [comment]
![5](https://user-images.githubusercontent.com/11382521/34189910-d120a21a-e578-11e7-9827-62e5902a40f7.png)
![6](https://user-images.githubusercontent.com/11382521/34189923-f0c00778-e578-11e7-9776-1729b1ad28ad.png)

**Comment by guolindev** [comment]
I guess it's because your Book1.db get created at first time, but Book table not created. And when you modify any of your model structure, you need to increase the version number in litepal.xml, or LitePal won't execute upgrades.

So just change the version number into 2, I guess all the things will work fine.

**Comment by guolindev** [comment]
And you don't need to call Connector.getDatabase(). That's just for test, or you want create database ealiear than your first db operation.

**Comment by guolindev** [comment]
I'm sorry. I just retest for serval times, and I found that you can always use **val** in kotlin models. And LitePal would always work with that.

Your real problem is in the litepal.xml configuration file. The **mapping** spell is wrong. You missed one **p**
